acm/src/main/java/org/literacybridge/acm/config/DBConfiguration.java

```diff
@@ -272,6 +273,9 @@ public boolean shouldPreCacheWav() {
                 }
```

Comment:
Hmm I don't remember. I assume we just do it so that we have a least English if parsing fails...